### DISCUSSION TITLE
v2.15.0 TROJAN inside???

### DISCUSSION THREAD

**Discussion by jedrzejs3** [upscayl/upscayl#1083] [Q&A] [answered]
I have checked file, here is raport:
https://www.virustotal.com/gui/file/a3ddc7e1f022b45c4dd3df0e53917d7d02fd0194237a3b644f277f5a2c79aaac

**Comment by NayamAmarshe** [comment]
It's a false positive.

**Reply by jedrzejs3** [reply]
v2.11.0 don't have this alert:
https://www.virustotal.com/gui/file/29be0c61415a34db0c8ec55614185febb9f144d703dbd616fddaef5e9d3ecff3

Hmmm...

**Reply by NayamAmarshe** [reply]
It might be because I used Mac to compile the Windows build. The windows build was built inside a WINE Layer. It's a false positive for sure.

**Comment by aaronliu0130** [comment]
I don't see any positives in this report. Nearly all of them say "undetected".

**Comment by NayamAmarshe** [comment, marked as answer]
@jedrzejs3 You can download the EXE again, it should fix the issue :)
